api/valor_api/backend/metrics/detection.py

```diff
@@ -378,6 +405,11 @@ def _annotation_type_to_column(
                 grouper_mappings["label_id_to_grouper_id_mapping"],
                 value=models.GroundTruth.label_id,
             ).label("label_id_grouper"),
+            (
+                gfunc.ST_AsGeoJSON(
```

Comment:
`ST_AsGeoJSON` does not support the raster type. 

There are two options here...
1. Returning the original annotation will require using the same encoding we to load schemas.
2. Cast it to a envelope and only ever return bounding boxes.

Reply:
thanks for this callout! I used `ST_Envelope` on rasters in `detection.py`, and added tests to evaluate whether object detection works as expected for annotations with rasters.